single_joint_actuator:
    # Publish all joint states -----------------------------------
    joints_update:
      type: joint_state_controller/JointStateController
      publish_rate: 100
    
    # Position Controllers ---------------------------------------
    joint1_position_controller:
      type: effort_controllers/JointPositionController
      joint: joint1
      pid: {p: 0.01000, i: 0.0, d: 0.0, i_clamp_min: -10.0, i_clamp_max: 10.0, antiwindup: True}
      
    # Velocity Controllers ---------------------------------------
    joint1_velocity_controller:
      type: effort_controllers/JointVelocityController
      joint: joint1
      #pid: {p: 0.1, i: 0.0005, d: 0.0100, i_clamp_min: -100.0, i_clamp_max: 100.0, antiwindup: True}
      pid: {p: 0.1, i: 0.0005, d: 0.0100, i_clamp_min: -100.0, i_clamp_max: 100.0, antiwindup: True}
      
